app/realtime.php

```diff
@@ -32,41 +41,81 @@
 ini_set('default_socket_timeout', -1);
 Swoole\Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
 
-$server = new Server("0.0.0.0", 80);
+$server = new Server('0.0.0.0', 80);
+
+$server->set([
+    'websocket_compression' => true,
+    'package_max_length' => 81920
+]);
+
 $subscriptions = [];
 $connections = [];
 
-$server->on("workerStart", function ($server, $workerId) use (&$subscriptions, &$connections) {
-    Console::success('Worker '.++$workerId.' started succefully');
+$register = new Registry();
+
+$register->set('db', function () { // Register DB connection
```

Comment:
Is this being set here because of: `Swoole\Runtime::enableCoroutine(SWOOLE_HOOK_ALL);`?